Packs/Claroty/Integrations/Claroty/Claroty.yml

```diff
@@ -0,0 +1,491 @@
+      description: CVE publish date.
+      type: Date
+    - contextPath: Claroty.Asset.ClassType
+      description: The OT/IT class type.
+      type: String
+    - contextPath: Claroty.Asset.Criticality
+      description: The criticality of the asset, according to the Purdue model.
+      type: String
+    - contextPath: Claroty.Asset.FirmwareVersion
+      description: The FM version of the asset.
+      type: String
+    - contextPath: Claroty.Asset.HighestCVEScore
+      description: Highest CVE Score for the Asset.
+      type: Number
+    - contextPath: Claroty.Asset.IP
+      description: The IPv4 address of the asset.
+      type: String
+    - contextPath: Claroty.Asset.InsightName
+      description: The asset insight names generated by CTD.
+      type: String
+    - contextPath: Claroty.Asset.LastSeen
+      description: The date the asset was last seen.
+      type: Date
+    - contextPath: Claroty.Asset.MAC
+      description: The MAC address of the asset.
+      type: String
+    - contextPath: Claroty.Asset.Name
+      description: The asset name.
+      type: String
+    - contextPath: Claroty.Asset.ResourceID
+      description: The asset RID (AssetID-SiteID).
+      type: String
+    - contextPath: Claroty.Asset.RiskLevel
+      description: The risk indicator.
+      type: Number
+    - contextPath: Claroty.Asset.SiteID
+      description: The site ID of the asset.
+      type: Number
+    - contextPath: Claroty.Asset.SiteName
+      description: The site name of the asset.
+      type: String
+    - contextPath: Claroty.Asset.Vendor
+      description: The vendor of the asset.
+      type: String
+    - contextPath: Claroty.Asset.VirtualZone
+      description: The virtual zone of the asset.
+      type: String
+    - contextPath: Claroty.Asset.WasParsed
+      description: Whether the project was parsed.
+      type: String
+  - arguments:
+    - auto: PREDEFINED
+      default: false
+      defaultValue: all
+      description: Alert fields to return. The default value is "all".
+      isArray: false
+      name: fields
+      predefined:
+      - all
+      - resource_id
+      - type
+      - severity
+      - network_id
+      - resolved
+      - description
+      - alert_indicators
+      - actionable_assets
+      - category
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      defaultValue: timestamp
+      description: |-
+        The field by which to sort the results. The default value is "timestamp".
+        Default sort order is ascending
+      isArray: false
+      name: sort_by
+      predefined:
+      - resource_id
+      - type
+      - severity
+      - network_id
+      - resolved
+      - description
+      - alert_indicators
+      - actionable_assets
+      - id
+      - timestamp
+      required: false
+      secret: false
+    - default: false
+      description: Returns alerts that match this alert type.
+      isArray: false
+      name: type
```

Comment:
can we add predefined values here?

Reply:
We change the value in the system often so we can't predefine values